frankly034/Dantown
# #168428994 add user login

#### What does this PR do?
Setup login user login feature 

#### Description of Task to be completed?
- add login controller
- add login route
- add login validation
- add login tests

#### How should this be manually tested?
* clone the repo
* run `npm install`
* start the app with `npm run start:dev`
* visit `POST` `localhost:3000/api/v1/auth/login` 
* Enter a valid email and password
* You should get a `200` status on success or `400` on failure

#### Any background context you want to provide?
* must have `node` installed

#### What are the relevant pivotal tracker stories?
[Pivotal tracker story #168428994](https://www.pivotaltracker.com/n/projects/2385884/stories/168428994)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
